explorer/explorer.go

```diff
@@ -19,13 +19,15 @@ import (
 	"time"
 
 	"github.com/decred/dcrd/chaincfg"
-	"github.com/decred/dcrd/dcrjson/v2"
+	dcrjson "github.com/decred/dcrd/dcrjson/v2"
```

Comment:
Is a linter complaining about this import without the name?  With modules enabled, you don't need to name it as the v2 is ignored.